src/bp/ui-studio/src/web/components/Layout/Toolbar/style.scss

```diff
@@ -11,7 +11,6 @@
 
 .toolbar {
   background: #fff;
-  border-bottom: 1px solid var(--c-neutral--light-2);
```

Comment:
Should leave this there until the whole redesign is completed since there's a lack of contrast with some modules otherwise